src/settings.js

```diff
@@ -24,7 +24,8 @@ export default {
   BINARIES_DIRECTORY: path.join(__dirname, '../data/binaries'),
   DEFAULT_EVENT_TTL: 60,
   DEVICE_DIRECTORY: path.join(__dirname, '../data/deviceKeys'),
-  SERVER_CONFIG: {
+  TCP_DEVICE_SERVER_CONFIG: {
+    ENABLE_SYSTEM_FIRWMARE_AUTOUPDATES: true,
```

Comment:
Why is this under `TCP_DEVICE_SERVER_CONFIG` and not at a higher level? It doesn't have anything to do with those configuration values.  Those are used in the tcp server -- this is just used in the logic of the protocol.